packages/lib-classifier/src/store/AnnotatedSteps/AnnotatedSteps.js

```diff
@@ -133,11 +133,21 @@ const AnnotatedSteps = types.model('AnnotatedSteps', {
     })
   }
 
+  /** Checks if the user has started making any annotations. Returns boolean */
+  function checkForProgress() {
+    let progressFlag = false
+    self.annotations.forEach(annotation => {
+      progressFlag ||= annotation.inProgress
```

Comment:
I think you can check `annotation._inProgress` directly here, which saves another view.